### PR TITLE
Tcl/Tk 8.6.10

### PR DESCRIPTION
**Pull request by Mno-hime**
Updated Tcl/Tk packages to their latest respective releases (modulo `tcl-sqlite` which I updated to `database/sqlite-3` version).

Testing done:
* Tcl tested with Pgtcl
* Pgctl tested via it's testsuite, two enhancements identified: https://github.com/flightaware/Pgtcl/issues/27 and https://github.com/flightaware/Pgtcl/issues/28
* Tk tested via `gitk` and `xorriso`
* tcllib & tklib tested via their respective test suites (though those test suites' results weren't particularly clear to me)
* tcl-sqlite wasn't tested

### CODE REVIEW

**Comment by Mno-hime** [comment]
Tcl 8.6.9 causes fails like this in SQLite's TCL Tests:
```
! fts5aa-full-1.1 expected: [
]
! fts5aa-full-1.1 got:      []
```
~~Filed: https://core.tcl.tk/tcl/tktview/2c154a40be9e1dd81694485ed9558a9eb831f55b~~
Ticked moved to: https://core.tcl.tk/tdbcsqlite3/info/145df89159588c56.

It was already reported and fixed as part of https://core.tcl.tk/tcl/info/cc1e91552c, will be in Tcl 8.6.10.

**Comment by Mno-hime** [comment]
> It was already reported and fixed as part of https://core.tcl.tk/tcl/info/cc1e91552c, will be in Tcl 8.6.10.

Causes: https://core.tcl-lang.org/tcl/tktview/f9a45e58af22e444346e54b991dcd596a3a20883.

**Comment by Mno-hime** [comment]
> > It was already reported and fixed as part of https://core.tcl.tk/tcl/info/cc1e91552c, will be in Tcl 8.6.10.
> 
> Causes: https://core.tcl-lang.org/tcl/tktview/f9a45e58af22e444346e54b991dcd596a3a20883.

Tcl/Tk 8.6.10 got released last month, the problem above should be fixed there.

**Comment by Mno-hime** [comment]
> > > It was already reported and fixed as part of https://core.tcl.tk/tcl/info/cc1e91552c, will be in Tcl 8.6.10.
> > 
> > 
> > Causes: https://core.tcl-lang.org/tcl/tktview/f9a45e58af22e444346e54b991dcd596a3a20883.
> 
> Tcl/Tk 8.6.10 got released last month, the problem above should be fixed there.

The Tcl/Tk 8.6.10 update indeed fixed this.

**Comment by alarcher** [comment]
@Mno-hime Do you want to merge it?

**Comment by Mno-hime** [comment]
> @Mno-hime Do you want to merge it?

@alarcher Looks fine for me.

**Comment by pyhalov** [comment]
@Mno-hime Can we merge this in one PR?

**Comment by Mno-hime** [comment]
> @Mno-hime Can we merge this in one PR?

You mean one commit? I thought we do one commit per component?

**Comment by pyhalov** [comment]
In one PR. I mean are there dependencies between these commits?

**Comment by pyhalov** [comment]
@Mno-hime Can we merge this PR as-is or are there any dependencies between modified components?

File: components/tcl/tk/Makefile [hunk 1]
```diff
@@ -21,35 +21,38 @@
...
 include ../../../make-rules/shared-macros.mk
 
-COMPONENT_NAME=		tk
-COMPONENT_VERSION=	8.6.7
-COMPONENT_MINOR_VERSION=	$(shell echo $(COMPONENT_VERSION) | $(GSED) -e 's/\([0-9]\+\)\.\([0-9]\+\)\.[0-9]\+/\1.\2/')
-COMPONENT_PROJECT_URL=	http://prdownloads.sourceforge.net/tk/
+COMPONENT_NAME=			tk
+COMPONENT_MAJOR_VERSION=	8.6
+COMPONENT_MINOR_VERSION=	10
+#COMPONENT_PATCH_VERSION=	.1
```
Comment (pyhalov) [inline components/tcl/tk/Makefile:34]: Should we remove COMPONENT_PATCH_VERSION (here and from COMPONENT_ARCHIVE?

**Pull request merged**